packages/next/src/next-devtools/dev-overlay/utils/save-devtools-config.ts

```diff
@@ -0,0 +1,47 @@
+import type { DevToolsConfig } from '../shared'
+import { devToolsConfigSchema } from '../../shared/devtools-config-schema'
+import { deepMerge } from '../../shared/deepmerge'
+
+let queuedPatch: DevToolsConfig = {}
```

Comment:
prefer to use a more specific name like `queuedConfigPatch` to tell what the patch is